app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,130 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.repository.TaskRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스의")
+class TaskServiceTest {
+    TaskService taskService;
+    final Long givenId = 0L;
+    final String givenTitle = "BJP";
```

Comment:
```suggestion
    private TaskService taskService;
    private final Long givenId = 0L;
    private final String givenTitle = "BJP";
```
nit: 접근제어자를 추가하는 것이 좋을 것 같아요!